packages/eslint-config-eslint/default.yml

```diff
@@ -36,30 +72,67 @@ rules:
     dot-notation: ["error", { allowKeywords: true }]
     eol-last: "error"
     eqeqeq: "error"
+
+    eslint-comments/disable-enable-pair: ["error", {allowWholeFile: true}]
```

Comment:
Just FYI, it's generally better practice to use `/* eslint foo:"off" */` to disable a rule for an entire file than to use the disable/enable comments around the entire file.  

I am adding an explanation about this here: https://github.com/mysticatea/eslint-plugin-eslint-comments/pull/60

Reply:
Agree

Reply:
Sounds good. Added a commit.